src/Symfony/Component/Config/Definition/Builder/NodeDefinition.php

```diff
@@ -169,6 +170,20 @@ public function isRequired()
     }
 
     /**
+     * Sets the node as deprecated.
+     *
+     * @param string $message Deprecation message
+     *
+     * @return $this
+     */
+    public function isDeprecated($message = 'The child node "%node%" at path "%path%" is deprecated.')
+    {
+        $this->deprecatedMessage = $message;
```

Comment:
[in Definition](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/DependencyInjection/Definition.php#L680), we enforce that the placeholder is used, we should do the same here IMHO, for both placeholders